cypress/integration/accordion.spec.js

```diff
@@ -35,4 +42,288 @@ describe( 'Accordion Block', () => {
+		cy.publish()
+		getAddresses( ( { currUrl, previewUrl } ) => {
+			cy.visit( previewUrl )
+			cy
+				.get( '.ugb-accordion' )
+				.invoke( 'attr', 'aria-expanded' )
+				.then( ariaExpanded => {
+					expect( ariaExpanded ).toBe( 'true' )
+					cy.visit( currUrl )
+				} )
+		} )
+
+		//Test 'Reverse arrow'
+		cy.openInspector( 'ugb/accordion', 'Style' )
+		cy.collapse( 'General' )
+		cy.adjust( 'Reverse arrow', true ).assertComputedStyle( {
+			'.ugb-accordion__heading': {
+				[ `flex-direction` ]: 'row-reverse',
+			},
+		} )
+
+		//Test General Alignment
+		const aligns = [ 'left', 'center', 'right' ]
+		aligns.forEach( align => {
+			cy.adjust( 'Align', align ).assertComputedStyle( {
+				'.ugb-inner-block': {
+					[ `text-align` ]: align,
+				},
+			} )
+		} )
+
+		cy.collapse( 'Container' )
+		//Test Single Background Color
+		cy.adjust( 'Background', {
+			[ `Color Type` ]: 'single',
+			[ `Background Color` ]: '#000000',
+			[ `Background Color Opacity` ]: '0.5',
+		} ).assertComputedStyle( {
+			'.ugb-accordion__heading': {
+				[ `background-color` ]: 'rgba(0, 0, 0, 0.5)',
+			},
+		} )
+
+		//Test Gradient Background Color
+		cy.collapse( 'Container' )
+		cy.adjust( 'Background', {
+			[ `Color Type` ]: 'gradient',
+			[ `Background Color #1` ]: '#f00069',
+			[ `Background Color #2` ]: '#000000',
+			[ `Adv. Gradient Color Settings` ]: {
+		  [ `Gradient Direction (degrees)` ]: '180deg',
+		  [ `Color 1 Location` ]: '11%',
+		  [ `Color 2 Location` ]: '80%',
+		  [ `Background Gradient Blend Mode` ]: 'hard-light',
+			},
+		} ).assertComputedStyle( {
+			'.ugb-accordion__heading:before': {
+				[ `background-image` ]: 'linear-gradient(#f00069 11%, #000000 80%)',
+				[ `mix-blend-mode` ]: 'hard-light',
+			},
+		} )
+
+		//Test Border Radius
+		cy.adjust( 'Border Radius', 30 ).assertComputedStyle( {
+			'.ugb-accordion__heading': {
+				[ `border-radius` ]: '30px',
+			},
+		} )
+
+		//Test Border Options
+		cy.adjust( 'Borders', 'solid' )
+		cy.adjust( 'Border Width', 3 )
+		cy.adjust( 'Border Color', '#f1f1f1' ).assertComputedStyle( {
+			'.ugb-accordion__heading': {
+				[ `border-style` ]: 'solid',
+				[ `border-color` ]: '#f1f1f1',
+				[ `border-top-width` ]: '3px',
+				[ `border-bottom-width` ]: '3px',
+				[ `border-left-width` ]: '3px',
+				[ `border-right-width` ]: '3px',
+			},
+		} )
+
+		//Test Shadow / Outline
+		cy.adjust( 'Shadow / Outline', 9 )
+			.assertClassName( '.ugb-accordion__heading', 'ugb--shadow-9' )
+
+		cy.collapse( 'Spacing' )
+		// Test Padding options
+		cy.adjust( 'Padding', 46 ).assertComputedStyle( {
+			'.ugb-accordion__heading': {
+				[ `padding-top` ]: '46px',
+				[ `padding-right` ]: '46px',
+				[ `padding-bottom` ]: '46px',
+				[ `padding-left` ]: '46px',
+			},
+		} )
+
+		//Test Title spacing
+		cy.adjust( 'Title', 63 )
+		cy.typeBlock( 'ugb/accordion', '.ugb-accordion__title', `Accordion 1` )
+			.assertComputedStyle( {
+				'.ugb-accordion--open': {
```

Comment:
this is not the right selector for the title spacing. `.ugb-accordion--open` is only used for collapsed accordion